src/main/java/frc/robot/RobotContainer.java

```diff
@@ -54,22 +56,26 @@
 
 
   // subsystems
+
+  private static ClimbSubsystem climbSubsystem;
   private static DriveBaseSubsystem driveBaseSubsystem;
   private static CDSSubsystem CDSSubsystem;
   private static IntakeSubsystem intakeSubsystem; 
   private static ShooterSubsystem shooterSubsystem;
   private static LimelightSubsystem limelightSubsystem;
 
   // commands
-  private final DriveBaseTeleopCommand driveBaseTeleopCommand = new DriveBaseTeleopCommand(driveBaseSubsystem);
-  private IntakeForwardCommand intakeForwardCommand = new IntakeForwardCommand(intakeSubsystem);
-  private IntakeReverseCommand intakeReverseCommand = new IntakeReverseCommand(intakeSubsystem);
+  private DriveBaseTeleopCommand driveBaseTeleopCommand;
```

Comment:
I made these changes so those commands would be initialized inside of the modular switch case.